support/dspace_proxy.py

```diff
@@ -31,5 +42,23 @@ def get(self, command, params=None, data=None):
         self.response = self.d.api_get(url, params, data)
         return self.response
 
+    def reauthenticate_loop(self):
+        while True:
+            # time.sleep(5*60)
+            stopevent.wait(5 * 60)  # every 5 minutes seems sufficient
+            if stopevent.is_set():
+                log("Reauthenticate loop is over", Severity.DEBUG)
+                break
+            self.reauthenticate()
+            log("Reauthenticating", Severity.DEBUG)
+
 
 rest_proxy = DspaceRESTProxy()
+reauth = threading.Thread(target=rest_proxy.reauthenticate_loop, daemon=True)
+stopevent = threading.Event()
+reauth.start()
```

Comment:
This reauthentication is solved in the `client.py` Please turn off this reauthentication by the cfg

Reply:
done. Please merge